api/api/views/media_views.py

```diff
@@ -26,7 +27,7 @@ class MediaViewSet(ReadOnlyModelViewSet):
     pagination_class = StandardPagination
 
     # Populate these in the corresponding subclass
-    model_class = None
+    model_class: AbstractMedia = None
```

Comment:
This takes a subclass of `AbstractMedia`, not an instance. I think this is the right type.

```suggestion
    model_class: Type[AbstractMedia] = None
```

Reply:
Oops, missed this, thank you!

Reply:
Fixed in the last commit, also TIL that the regular `type` builtin works for `Type` in this context as well :tada: Ruff automatically changed that for me :sweat_smile: